birdbox/birdbox/settings/base.py

```diff
@@ -237,17 +238,13 @@ def path_from_root(*args):
 MEDIA_ROOT = os.path.join(BIRDBOX_BASE_DIR, "media")
 MEDIA_URL = "/media/"
 
-
-def set_whitenoise_headers(headers, path, url):
-    if "/fonts/" in url:
-        headers["Cache-Control"] = "public, max-age=604800"  # one week
```

Comment:
Why remove this?

Reply:
Mentioned in person but for the record: fonts in Birdbox come from the /static/ path, which is already automatically given a long cache lease by whitenoise, so this was actually working against that and didn't give us a benefit.